main.typ

```diff
@@ -40,6 +40,20 @@
 
 = Typical Open Source Development
 
+Open source software development is a collaborative effort between developers and users. The users of the software are able to contribute to the development of the software by reporting bugs, suggesting features, and even contributing code. The developers of the software are able to use the feedback from the users to improve the software. Open source software development differs from traditional software development in two major ways. The first being that the users are able to contribute to the development of the software. The second being that almost all collaboration between developers happens asynchronously, through the internet.
+
+Often times, contributors to open source software development are completely anonymous. This is because the contributors are not required to provide any personal information when contributing to the software. This is a major difference from traditional software development, where the developers are usually working in the same location and are able to communicate face to face. It creates a completely different dynamic when managing a project.
+
+== Project Management in Open Source Software
+
+For many smaller open source projects, the project is managed by a "Benevolent Dictator For Life", where the goals and priorities of the project are set by a single person. This same person is also responsible for making the final decisions on what features are added to the project. They take ownership of the code, and are completely responsible for it. This is a very simple way to manage a project, but it does not scale well. As the project grows, it becomes more difficult for a single person to manage the project. In mid-sized open source projects, the project may be managed by a small group of people, but under the same principles as the "Benevolent Dictator For Life".
+
+That's not to say that all open source projects are managed this way. Some open source projects are managed by a company, where the company is responsible for the development of the project. In this case, the company is responsible for the management of the project, and the developers are responsible for the development of the project. This is similar to traditional software development, but the developers are still able to contribute to the project. However, in this management style, the developers generally have an equal say in what code is merged into the project.
+
+That being said, there are many large and widely used open source projects that are still ultimately managed by a Benevolent Dictator. The Linux kernel is a great example of this: Linus Torvalds is the Benevolent Dictator For Life, but under him are a few layers of maintainers that oversee different parts of the kernel. These maintainers are responsible for reviewing and vouching for the quality of the code that is submitted up the chain of command, eventually to Linus Torvalds. #cite("user229044_2010") #cite("How-Linux-is-Built")
```

Comment:
- cut "that being said". (try to remove any language that sounds like you are speaking)
- cut "is a great example of this" ==> that is subjective, allow the reader to determine if its great or not just by writing the example